templates/app-extra.yaml

```diff
@@ -28,7 +28,7 @@ Resources:
       ListenerArn: !Ref Listener443Arn
       Certificates:
         - CertificateArn: !ImportValue
-            'Fn::Sub': '${AWS::Region}-synapselogin-shared-SSLCertificateSageIT'
+            'Fn::Sub': '${AWS::Region}-synapse-login-scipooldev-sageit-cert-SSLCertificateSageIT'
   HttpRedirectRule:
     Type: AWS::ElasticLoadBalancingV2::ListenerRule
     Properties:
```

Comment:
Is that not going to fail when we deploy on prod?

Reply:
your right. lemme fix